src/components/common/DocumentsDelivery.vue

```diff
@@ -109,6 +113,14 @@ export default class DocumentsDelivery extends Mixins(CommonMixin) {
     return (this.validate && !this.getFlagsReviewCertify.isValidDocumentOptionalEmail)
   }
 
+  get userEmail (): string {
+    return this.applicantEmail ? this.applicantEmail : this.getUserEmail
+  }
+
+  get emailLabel (): string {
+    return this.getResource.userEmailLabel ? this.getResource.userEmailLabel : 'Completing Party'
+  }
```

Comment:
compute email label. It is different for other filings and Restorations

Reply:
Could also do `return (a || b)` here, fwiw.